src/make_trade.ts

```diff
@@ -1,4 +1,4 @@
-import { concat, hexlify, hexValue } from "@ethersproject/bytes";
+import { concat, hexlify, hexValue, hexZeroPad } from "@ethersproject/bytes";
```

Comment:
These are re-exported by ethers and should probably be used from there. You could:
```ts
const { concat, hexlify, hexValue, hexZeroPad } = ethers.utils;
```

The issue is that this depends on a transitive dependency being there, which is more brittle than using the specifically defined one.